angular-workspace/projects/ni/nimble-angular/table-column/anchor/tests/nimble-table-column-anchor-router-link-with-href.directive.spec.ts

```diff
@@ -124,4 +125,138 @@ describe('Nimble anchor table column RouterLinkWithHrefDirective', () => {
             expect(routerNavigateByUrlSpy).not.toHaveBeenCalled();
         }));
     });
+
+    describe('handleLinkClick', () => {
```

Comment:
In addition to adding unit tests, I think it would be prudent to test this in an actual app (or possibly multiple apps with different `baseHref` configurations) so that we can have a higher level of confidence the integration works as expected.

Reply:
I'm not sure I know how to test this in an app. I can't easily swap in the Nimble table for an existing one. Would I just insert a new table and hard-code some data with URLs that look like the ones being used in the actual table? How did you test this initially to find out it wasn't working correctly?